runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java

```diff
@@ -122,10 +123,11 @@ public Dimension render(Graphics2D graphics)
 		}
 
 		//Widget bank = client.getWidget(WidgetInfo.BANK_ITEM_CONTAINER);
-		//if (bank != null && !bank.isHidden())
-		//{
-		//	return null;
-		//}
+		Widget bank_inv = client.getWidget(WidgetInfo.BANK_INVENTORY);
```

Comment:
Nitpick: use [camel case](https://en.wikipedia.org/wiki/Camel_case) for `bank_inv` to stay consistent.